app/src/main/AndroidManifest.xml

```diff
@@ -36,7 +38,7 @@
 
 		<meta-data
 			android:name="com.google.android.geo.API_KEY"
-			android:value="@string/google_maps_key" />
+			android:value="https://www.google.com" />
```

Comment:
Is there a reason why the `API_KEY` is `google.com`?

Reply:
Fixed it!